Form/Type/SearchType.php

```diff
@@ -17,7 +19,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'term',
-                'text',
+                TextType::class,
```

Comment:
In composer.json staan geen PHP-versie (meer) vermeld. Ik vind dat ook compleet achterlijk.
In README.md staat het ook niet
Heb het in 0e284d79a62ee8e76eed5309bb7d17a509f9e9d0 uit travis-config gehaald ;-)
In onze mails naar klanten staat 5.5 ook als minimum